src/compiler/Uno.Compiler.Core/Syntax/UxlProcessor.cs

```diff
@@ -619,10 +620,16 @@ void WriteImageFile(ImageFile f)
 
                 using (var originalImg = Image.FromFile(src))
                 {
+                    if (_env.IsDefined("iOS") && Image.IsAlphaPixelFormat(originalImg.PixelFormat) )
+                    {
+                        var source = new Source(src);
+                        Log.Warning(source, ErrorCode.E0000, "iOS App Store doesn't accept Images with transparency.");
```

Comment:
Now you're no longer calling `IfImageContainsTransparent()` here... I think that's completely fine, but then we should also drop adding the `IfImageContainsTransparent()`-method...

Reply:
Removed and everything was squashed